src/app/(BottomTabNavigation)/AllCases/CaseScreen/styles.ts

```diff
@@ -14,6 +14,7 @@ export default StyleSheet.create({
   scrollView: {
     flex: 1,
     alignItems: 'center',
+    minHeight: 900,
```

Comment:
This fixes`ScrollView` issue but wanted to know where this number comes from and if we can somehow confirm that this will work across all screen types

Reply:
I honestly made up a number that worked when I was testing and forgot about it. I'm not sure how to determine this width necessarily. Although, while looking through the `FlatList` documentation online, I found a prop called `getItemLayout` that allows you to set the height of a flat list if you know the height of the components. We don't currently use a FlatList (only a ScrollView), but perhaps this could be used to solve the issue (?)

FlatList docs: https://reactnative.dev/docs/flatlist#getitemlayout

Reply:
Does `FlatList` work well with different components? I've only seen it used when rendering the same component multiple times, like the `AllCases` screen. Looked through CNSC as well and we used `flexGrow` with `ScrollView` so we can also explore that